fs/operations/interrupt.go

```diff
@@ -0,0 +1,22 @@
+//+build !windows
```

Comment:
These should probably go in `interrupt_test.go` as they are only used in the tests aren't they?